proto/trisa/gds/models/v1beta1/models.proto

```diff
@@ -103,12 +110,12 @@ message Contact {
     // Phone number is optional, but it is strongly suggested
     string phone = 3;
 
-    // Token for email verification
-    bool verified = 4;
-    string token = 5;
-
     // Optional KYC data if required for the directory service contact.
-    ivms101.NaturalPerson person = 6;
+    ivms101.NaturalPerson person = 4;
+
+    // Extra data that might be stored by the directory service related to the Contact.
+    // Should not be populated by client code.
+    google.protobuf.Any extra = 5;
```

Comment:
Added an "extra" field so directory service could store arbitrary data for operations, e.g. verification tokens.